packages/framework-types/src/concepts/register.ts

```diff
@@ -40,4 +44,9 @@ export class Register {
     this.eventList.push(...events)
     return this
   }
+
+  public async flush(config: BoosterConfig): Promise<void> {
```

Comment:
We can make this function not to need the `config` and have a better UX by doing any of the following things:
1. Make the constructor of this class receive the config and store as instance member. Then pass it to the flusher here.
2. Make the `flusher`  to already have the config stored in the function closure.

Maybe point 1 is simpler

Reply:
I see you point, but there are some considerations:

1. For the option 1. The Register is instantiated several times in different places and not all of them has a config object. This will force us to pass the `config` object for all the dependency tree until we reach the `new Register` method or to refactor a lot of classes.  In my opinion the `flush` call is not something that we expect to be used so much, so adding a config object is not a problem.
2. For the option 2. Not sure if I understood you, `handle` is a static method. What do you mean with `stored in the closure` ??